src/ralph/api/routers/health.py

```diff
@@ -6,19 +6,18 @@
 from fastapi import APIRouter, status
 from fastapi.responses import JSONResponse
 
-from ralph.backends.conf import backends_settings
+from ralph.backends.loader import get_lrs_backends
 from ralph.backends.lrs.base import BaseAsyncLRSBackend, BaseLRSBackend
 from ralph.conf import settings
-from ralph.utils import await_if_coroutine, get_backend_instance
+from ralph.utils import await_if_coroutine, get_backend_class
 
 logger = logging.getLogger(__name__)
 
 router = APIRouter()
 
-BACKEND_CLIENT: Union[BaseLRSBackend, BaseAsyncLRSBackend] = get_backend_instance(
-    backend_type=backends_settings.BACKENDS.LRS,
-    backend_name=settings.RUNSERVER_BACKEND,
-)
+BACKEND_CLIENT: Union[BaseLRSBackend, BaseAsyncLRSBackend] = get_backend_class(
+    backends=get_lrs_backends(), name=settings.RUNSERVER_BACKEND
+)()
```

Comment:
Good question) We could make the `get_backend_class` accept just a name and search for all backends instead - however, then it would do a bit more work than necessary to retrieve an LRS backend and might return another backend type instead (as data and lrs backends share the same `name` attribute).

Alternatively we could implement a dedicated one-line-long `get_lrs_backend(name)` method :
```python
return get_backend_class(get_lrs_backends(), name=name)()
```
Or maybe just `get_lrs_backend()` ?
```python
return get_backend_class(get_lrs_backends(), name=settings.RUNSERVER_BACKEND)()
```
WDYT?

Reply:
I am not sure I understand what you are proposing. What I had in mind was something like:
```
def get_backend_class(name: str) -> Any:
    """Return the backend class from available backends by its name."""
    backends=get_lrs_backends()
    backend_class = backends.get(name)
 ```
 But it is a very minor issue in any case so don't bother too much.

Reply:
Thanks for clarifying) I think the problem is that we use the `get_backend_class` method not only for lrs backends but also for data and stream backends (in `cli.py`) where we have:
```
get_backend_class(get_cli_backends(), backend)
get_backend_class(get_cli_write_backends(), backend)
get_backend_class(get_cli_list_backends(), backend)
```

Reply:
Ah yes, ok it makes sense. Thank you.